server/models/Tv.ts

```diff
@@ -141,7 +141,7 @@ export const mapSeasonWithEpisodes = (
   season: TmdbSeasonWithEpisodes
 ): SeasonWithEpisodes => ({
   airDate: season.air_date,
-  episodeCount: season.episode_count,
+  episodeCount: season.episodes.length,
```

Comment:
Ah, I don't think we should be using the length instead of the value returned by TMDB here. The number of planned episodes is useful information even if we don't have the episode data yet.

Reply:
The `episode_count` isn't actually returned by the API call for the season details. And the planned episodes are present in the `episodes` field.

Reply:
Regardless, I don't think it is good to manipulate this data here. This should be the data from TMDB...

Reply:
Gotcha, will remove this in this case 👍🏼